elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "miniBill/elm-avataaars",
     "summary": "A library for rendering cute SVG avatars, art by Pablo Stanley",
     "license": "MIT",
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
I don't know if this is an issue that I manually bumped this, I hope it doesn't cause issue for you when you make the release (😃🤞🏻).

Reply:
In general you should never bump the version number in PRs, because the author may have several pending, or want to release later, or similar.

Reply:
Okay, I'll bump it back down 👌🏻

Reply:
Bumbed the version number back down.